ebml_matroska.xml

```diff
@@ -332,7 +332,21 @@
   <element name="BlockAddIDType" path="\Segment\Tracks\TrackEntry\BlockAdditionMapping\BlockAddIDType" id="0x41E7" type="uinteger" minOccurs="1" maxOccurs="1" minver="4" default="0">
     <documentation lang="en" purpose="definition">Stores the registered identifer of the Block Additional Mapping to define how the BlockAdditional data should be handled.</documentation>
     <extension webm="0"/>
-  </element>
+    <restriction>
+      <enum value="0" label="BlockAddIDValue defines the content type">
+        <documentation lang="en" purpose="definition">This value indicates that BlockAddIDValue defines the content type, and a list of content type is in BlockAddID documentation.</documentation>
+      </enum>
+      <enum value="1" label="Codec specific content">
+        <documentation lang="en" purpose="definition">This value indicates that BlockAddIDExtraData defines the content type, and this definition depends on the Codec ID used. A transcoder SHOULD discard this content if the addition is not supported</documentation>
+      </enum>
+      <enum value="2" label="General purpose content">
+        <documentation lang="en" purpose="definition">This value indicates that BlockAddIDExtraData defines the content type, and this definition does not depend on the Codec ID used. A transcoder SHOULD NOT discard this content if the addition is not supported</documentation>
```

Comment:
Same argument here. I'd rather the whole sentence about keeping/discarded din't exist.